Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -147,45 +151,88 @@ class TouchIDSetting: Setting {
         // we can disable interaction of the switch and still handle tap events.
         let control = UISwitch()
         control.onTintColor = UIConstants.ControlTintColor
-        control.on = authInfo?.useTouchID ?? false
+        self.authInfo = KeychainWrapper.authenticationInfo()
```

Comment:
Mainly for ease of use in accessing that information from the child classes, which don't invalidate the data every time they access it. But I'll refactor it a bit.